internal/ingress/annotations/proxyssl/main.go

```diff
@@ -32,15 +32,15 @@ import (
 
 const (
 	defaultProxySSLCiphers     = "DEFAULT"
-	defaultProxySSLProtocols   = "TLSv1 TLSv1.1 TLSv1.2"
+	defaultProxySSLProtocols   = "TLSv1.2"
 	defaultProxySSLVerify      = "off"
 	defaultProxySSLVerifyDepth = 1
 	defaultProxySSLServerName  = "off"
 )
 
 var (
 	proxySSLOnOffRegex    = regexp.MustCompile(`^(on|off)$`)
-	proxySSLProtocolRegex = regexp.MustCompile(`^(SSLv2|SSLv3|TLSv1|TLSv1\.1|TLSv1\.2|TLSv1\.3| )*$`)
+	proxySSLProtocolRegex = regexp.MustCompile(`^(TLSv1\.2|TLSv1\.3| )*$`)
```

Comment:
Is this being used for validating user defined versions? If so, I would not narrow it and still allow users to manually enable older versions by keeping this untouched.

Reply:
@gacko, can you run a curl with older TLS using controller v1.10.1 on your kind/minikube and advise what happens

Reply:
Ah, now I get you. It's not even supported image-wise! I thought we are removing it for security reasons. 😅